app/models/offer.rb

```diff
@@ -87,7 +87,14 @@ def set_initial_state
       offer.received_at = Time.now
     end
 
-    after_transition :on => :submit, :do => :send_new_offer_notification
+    after_transition :on => :submit do |offer, transition|
```

Comment:
Unused block argument - `transition`. If it's necessary, use `_` or `_transition` as an argument name to indicate that it won't be used.<br>Use the new Ruby 1.9 hash syntax.